### ISSUE TITLE
关于评估

### ISSUE THREAD

**Issue by zhaoloulou** [PaddlePaddle/PaddleOCR#4466]
[2021/10/27 17:03:39] root INFO: Architecture : 
[2021/10/27 17:03:39] root INFO:     Backbone : 
[2021/10/27 17:03:39] root INFO:         model_name : large
[2021/10/27 17:03:39] root INFO:         name : MobileNetV3
[2021/10/27 17:03:39] root INFO:         scale : 0.5
[2021/10/27 17:03:39] root INFO:     Head : 
[2021/10/27 17:03:39] root INFO:         fc_decay : 0
[2021/10/27 17:03:39] root INFO:         name : CTCHead
[2021/10/27 17:03:39] root INFO:     Neck : 
[2021/10/27 17:03:39] root INFO:         encoder_type : rnn
[2021/10/27 17:03:39] root INFO:         hidden_size : 96
[2021/10/27 17:03:39] root INFO:         name : SequenceEncoder
[2021/10/27 17:03:39] root INFO:     Transform : None
[2021/10/27 17:03:39] root INFO:     algorithm : CRNN
[2021/10/27 17:03:39] root INFO:     model_type : rec
[2021/10/27 17:03:39] root INFO: Eval : 
[2021/10/27 17:03:39] root INFO:     dataset : 
[2021/10/27 17:03:39] root INFO:         data_dir : ./train_data/ic15_data
[2021/10/27 17:03:39] root INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_test.txt']
[2021/10/27 17:03:39] root INFO:         name : SimpleDataSet
[2021/10/27 17:03:39] root INFO:         transforms : 
[2021/10/27 17:03:39] root INFO:             DecodeImage : 
[2021/10/27 17:03:39] root INFO:                 channel_first : False
[2021/10/27 17:03:39] root INFO:                 img_mode : BGR
[2021/10/27 17:03:39] root INFO:             CTCLabelEncode : None
[2021/10/27 17:03:39] root INFO:             RecResizeImg : 
[2021/10/27 17:03:39] root INFO:                 image_shape : [3, 32, 100]
[2021/10/27 17:03:39] root INFO:             KeepKeys : 
[2021/10/27 17:03:39] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/10/27 17:03:39] root INFO:     loader : 
[2021/10/27 17:03:39] root INFO:         batch_size_per_card : 256
[2021/10/27 17:03:39] root INFO:         drop_last : False
[2021/10/27 17:03:39] root INFO:         num_workers : 4
[2021/10/27 17:03:39] root INFO:         shuffle : False
[2021/10/27 17:03:39] root INFO:         use_shared_memory : False
[2021/10/27 17:03:39] root INFO: Global : 
[2021/10/27 17:03:39] root INFO:     cal_metric_during_train : True
[2021/10/27 17:03:39] root INFO:     character_dict_path : ppocr/utils/ppocr_keys_v1.txt
[2021/10/27 17:03:39] root INFO:     character_type : ch
[2021/10/27 17:03:39] root INFO:     checkpoints : None
[2021/10/27 17:03:39] root INFO:     debug : False
[2021/10/27 17:03:39] root INFO:     distributed : True
[2021/10/27 17:03:39] root INFO:     epoch_num : 72
[2021/10/27 17:03:39] root INFO:     eval_batch_step : [0, 2000]
[2021/10/27 17:03:39] root INFO:     infer_img : doc/imgs_words_en/word_10.png
[2021/10/27 17:03:39] root INFO:     infer_mode : False
[2021/10/27 17:03:39] root INFO:     log_smooth_window : 20
[2021/10/27 17:03:39] root INFO:     max_text_length : 25
[2021/10/27 17:03:39] root INFO:     pretrained_model : model/ch_ppocr_server_v2.0_rec_pre/best_accuracy
[2021/10/27 17:03:39] root INFO:     print_batch_step : 10
[2021/10/27 17:03:39] root INFO:     save_epoch_step : 3
[2021/10/27 17:03:39] root INFO:     save_inference_dir : ./
[2021/10/27 17:03:39] root INFO:     save_model_dir : ./output/rec/ic15/
[2021/10/27 17:03:39] root INFO:     save_res_path : ./output/rec/predicts_ic15.txt
[2021/10/27 17:03:39] root INFO:     use_gpu : False
[2021/10/27 17:03:39] root INFO:     use_space_char : False
[2021/10/27 17:03:39] root INFO:     use_visualdl : False
[2021/10/27 17:03:39] root INFO: Loss : 
[2021/10/27 17:03:39] root INFO:     name : CTCLoss
[2021/10/27 17:03:39] root INFO: Metric : 
[2021/10/27 17:03:39] root INFO:     main_indicator : acc
[2021/10/27 17:03:39] root INFO:     name : RecMetric
[2021/10/27 17:03:39] root INFO: Optimizer : 
[2021/10/27 17:03:39] root INFO:     beta1 : 0.9
[2021/10/27 17:03:39] root INFO:     beta2 : 0.999
[2021/10/27 17:03:39] root INFO:     lr : 
[2021/10/27 17:03:39] root INFO:         learning_rate : 0.0005
[2021/10/27 17:03:39] root INFO:     name : Adam
[2021/10/27 17:03:39] root INFO:     regularizer : 
[2021/10/27 17:03:39] root INFO:         factor : 0
[2021/10/27 17:03:39] root INFO:         name : L2
[2021/10/27 17:03:39] root INFO: PostProcess : 
[2021/10/27 17:03:39] root INFO:     name : CTCLabelDecode
[2021/10/27 17:03:39] root INFO: Train : 
[2021/10/27 17:03:39] root INFO:     dataset : 
[2021/10/27 17:03:39] root INFO:         data_dir : ./train_data/ic15_data/
[2021/10/27 17:03:39] root INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_train.txt']
[2021/10/27 17:03:39] root INFO:         name : SimpleDataSet
[2021/10/27 17:03:39] root INFO:         transforms : 
[2021/10/27 17:03:39] root INFO:             DecodeImage : 
[2021/10/27 17:03:39] root INFO:                 channel_first : False
[2021/10/27 17:03:39] root INFO:                 img_mode : BGR
[2021/10/27 17:03:39] root INFO:             CTCLabelEncode : None
[2021/10/27 17:03:39] root INFO:             RecResizeImg : 
[2021/10/27 17:03:39] root INFO:                 image_shape : [3, 32, 100]
[2021/10/27 17:03:39] root INFO:             KeepKeys : 
[2021/10/27 17:03:39] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/10/27 17:03:39] root INFO:     loader : 
[2021/10/27 17:03:39] root INFO:         batch_size_per_card : 256
[2021/10/27 17:03:39] root INFO:         drop_last : True
[2021/10/27 17:03:39] root INFO:         num_workers : 8
[2021/10/27 17:03:39] root INFO:         shuffle : True
[2021/10/27 17:03:39] root INFO:         use_shared_memory : False
[2021/10/27 17:03:39] root INFO: train with paddle 2.1.3 and device CPUPlace
[2021/10/27 17:03:39] root INFO: Initialize indexs of datasets:['./train_data/ic15_data/rec_gt_test.txt']
[2021/10/27 17:03:40] root INFO: The shape of model params backbone.conv1.conv.weight [8, 3, 3, 3] not matched with loaded params backbone.conv1_1._conv.weight [32, 3, 3, 3] !
[2021/10/27 17:03:40] root INFO: The shape of model params backbone.conv1.bn.weight [8] not matched with loaded params backbone.conv1_1._batch_norm.weight [32] !
[2021/10/27 17:03:40] root INFO: The shape of model params backbone.conv1.bn.bias [8] not matched with loaded params backbone.conv1_1._batch_norm.bias [32] !

eval model::   0%|          | 0/28 [00:00<?, ?it/s]
eval model::   4%|▎         | 1/28 [00:03<01:26,  3.20s/it]
eval model::   7%|▋         | 2/28 [00:05<01:16,  2.95s/it]
eval model::  11%|█         | 3/28 [00:08<01:11,  2.84s/it]
eval model::  14%|█▍        | 4/28 [00:11<01:06,  2.76s/it]
eval model::  18%|█▊        | 5/28 [00:13<01:02,  2.71s/it]
eval model::  21%|██▏       | 6/28 [00:16<00:59,  2.69s/it]
eval model::  25%|██▌       | 7/28 [00:19<00:56,  2.68s/it]
eval model::  29%|██▊       | 8/28 [00:21<00:53,  2.69s/it]
eval model::  32%|███▏      | 9/28 [00:24<00:52,  2.74s/it]
eval model::  36%|███▌      | 10/28 [00:28<00:53,  2.95s/it]
eval model::  39%|███▉      | 11/28 [00:31<00:52,  3.10s/it]
eval model::  43%|████▎     | 12/28 [00:35<00:51,  3.22s/it]
eval model::  46%|████▋     | 13/28 [00:38<00:48,  3.26s/it]
eval model::  50%|█████     | 14/28 [00:41<00:45,  3.25s/it]
eval model::  54%|█████▎    | 15/28 [00:44<00:40,  3.14s/it]
eval model::  57%|█████▋    | 16/28 [00:47<00:35,  3.00s/it]
eval model::  61%|██████    | 17/28 [00:49<00:31,  2.89s/it]
eval model::  64%|██████▍   | 18/28 [00:52<00:28,  2.82s/it]
eval model::  68%|██████▊   | 19/28 [00:55<00:25,  2.80s/it]
eval model::  71%|███████▏  | 20/28 [00:57<00:21,  2.74s/it]
eval model::  75%|███████▌  | 21/28 [01:00<00:19,  2.75s/it]
eval model::  79%|███████▊  | 22/28 [01:03<00:16,  2.72s/it]
eval model::  82%|████████▏ | 23/28 [01:06<00:13,  2.69s/it]
eval model::  86%|████████▌ | 24/28 [01:08<00:10,  2.70s/it]
eval model::  89%|████████▉ | 25/28 [01:11<00:08,  2.80s/it]
eval model::  93%|█████████▎| 26/28 [01:14<00:05,  2.92s/it]
eval model::  96%|█████████▋| 27/28 [01:18<00:03,  3.15s/it]
eval model:: 100%|██████████| 28/28 [01:21<00:00,  3.12s/it]
eval model:: 100%|██████████| 28/28 [01:21<00:00,  2.92s/it]
[2021/10/27 17:05:02] root INFO: metric eval ***************
[2021/10/27 17:05:02] root INFO: acc:0.0
[2021/10/27 17:05:02] root INFO: norm_edit_dis:0.0
[2021/10/27 17:05:02] root INFO: fps:92.01734841904383
INFO 2021-10-27 17:05:05,265 launch.py:268] Local processes completed.

你好，我下载你们提供的预训练模型用自己的数据评估，出现 问题：The shape of model params backbone.conv1.conv.weight [8, 3, 3, 3] not matched with loaded params backbone.conv1_1._conv.weight [32, 3, 3, 3] !

最后的acc也是0，请问是哪里参数配置有问题吗？谢谢！

**Comment by qqqqq127** [comment]
咋解决的
